common/lib/xmodule/xmodule/js/src/video/00_sjson.js

```diff
@@ -51,10 +61,44 @@ function() {
             return min;
         };
 
+        function filter(start, end) {
```

Comment:
Can you add some comments explaining what this function is supposed to do, and what arguments it's expected to accept? (Integers? Strings?)